cmd/billing.go

```diff
@@ -86,12 +86,12 @@ func activateWebhooks(ctx context.Context) error {
 		return fmt.Errorf("failed to get sdk from context: %w", err)
 	}
 
-	if _, err = sdk.Workspaces.SetFeatureFlags(ctx, shared.WorkspaceFeatureFlagRequest{
```

Comment:
tbh I think we should probably just remove this whole command and send everything through the dashboard. Add webhooks SDK feature through CLI is weird?

But I can do that in a separate PR after updating docs, etc.

Reply:
+1 and have an informational CTA to the billing page